components/ui/themes/DefaultTheme.ts

```diff
@@ -33,8 +33,8 @@ const Colors = {
   LightOrange: '#FDF1E6',
   GradientColors: ['#F59B4B', '#E86E04'],
   DisabledColors: ['#C7C7C7', '#C7C7C7'],
-  captureIconBorder: '#F59B4B',
   TimeoutHintBoxColor: '#FFF7E5',
+  TimeoutHintBoxBorder: '#FFF2D6',
```

Comment:
This border is not used in timeoutHintContainer for Default Theme?

Reply:
Done